lib/python/qmk/cli/flash.py

```diff
@@ -39,6 +41,7 @@ def print_bootloader_help():
 @cli.argument('filename', nargs='?', arg_only=True, type=qmk.path.FileType('r'), completer=FilesCompleter('.json'), help='The configurator export JSON to compile.')
 @cli.argument('-b', '--bootloaders', action='store_true', help='List the available bootloaders.')
 @cli.argument('-bl', '--bootloader', default='flash', help='The flash command, corresponding to qmk\'s make options of bootloaders.')
+@cli.argument('-t', '--type', help='The type of MCU. Required for HalfKay, HID and ISP boards.')
```

Comment:
```suggestion
@cli.argument('-m', '--mcu', help='The MCU name. Required for HalfKay, HID, USBAspLoader and ISP flashing.')
```

Reply:
I prefer `-m` as well, but wanted to be consistent with the current subcommands. I think only [`new-keyboard`](https://github.com/qmk/qmk_firmware/blob/master/lib/python/qmk/cli/new/keyboard.py#L186) asks for MCU and that uses `-t`.